src/js/common/components/RequiredLoginView.jsx

```diff
@@ -141,17 +99,17 @@ class RequiredLoginView extends React.Component {
       view = <LoadingIndicator setFocus message="Loading your information"/>;
     } else {
       if (this.props.authRequired === 1) {
-        if (this.state.accountType >= 1) {
+        if (this.props.userProfile.accountType >= 1) {
```

Comment:
Yeah, it's a little weird because it feels like you'd be making a constant that is renaming what is just a level number, but using something like `LOA_1` might make it clearer that `authRequired` and `accountType` are related information.